content/manager/delete-deployment.md

```diff
@@ -14,6 +14,15 @@ To remove the information related to a deployment on the Manager, run the follow
 cfy deployments delete nodecellar
 {{< /gsHighlight >}}
 
-# What's Next
+While you deleting a deployment you can also:
```

Comment:
The delete options are:

* -v, --verbose - Show verbose output. You can supply this up to three times, for example `-vvv`.
* -t, --tenant-name - Specify the tenant where the blueprint in stored. (Default: current tenant)
* -f, --force flag - Delete the deployment even it contains active nodes.